site/content/kapp/docs/develop/apply.md

```diff
@@ -67,6 +67,16 @@ Possible values: "" (default).
 
 Annotation value will be replaced with a unique ID on each deploy. This allows to force resource update as value changes every time.
 
+### kapp.k14s.io/renew-duration
+
+Available in v0.54.0+
+
+`kapp.k14s.io/renew-duration` annotation allows to set an interval at which you want your resource to get updated/renewed.
+
+Possible values: [ParseDuration](https://pkg.go.dev/time#ParseDuration).
+
+**Note** that for precise results use versioned resources or use always-replace update strategy which would rely on creationTimestamp of the resources.
```

Comment:
```suggestion
**Note**: For precise results use the `versioned resources` or `always-replace` update strategy which would rely on creationTimestamp of the resources.
```
Not sure whether we should mention `which would rely on creationTimestamp of the resources.`?

Reply:
I think we should not use `which would rely on creationTimestamp of the resources` as we are not adding implementation logic here how does kapp update resource internally.